tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -71,7 +71,7 @@ def handle_user(self, request, id_token, decoded_payload):
         email = decoded_payload["email"]
 
         # First account for the initial superuser
-        if (su_username := os.environ.get('DJANGO_SU_NAME')) and su_username == email:
+        if email == settings.DJANGO_SUPERUSER_NAME:
```

Comment:
Between the default in deployed contexts and the error that will get raised on startup if this is missing during local dev we don't need the extra null safety here anymore.